executor/executor.cc

```diff
@@ -421,6 +421,7 @@ int main(int argc, char** argv)
 			threads[i].cov.fd = kCoverFd + i;
 			cover_open(&threads[i].cov, false);
 			cover_protect(&threads[i].cov);
+			df_enable();
```

Comment:
Why is this added here? And why is it in this loop?

I think the way to check for capabilities is to first have syzkaller check if the feature exists. As an example, have a look at how KCSAN is being enabled (search for 'setup_kcsan' and look for uses of '/sys/kernel/debug/kcsan').

Reply:
Right, we do need to check for capabilities, this is an intermediate solution that will enable our feature unconditionally.
The intention is enable it on a per-task basis, therefore we want to call it on every thread. I don't think it works as intended right now though.

Reply:
yeah i just add this for testing to figure out what is wrong, it not the right way as i mention in the previous comment but i push it to let you see what i am trying to test